### PR TITLE
Start writing a better README

### LINKED ISSUE #12: Create a more welcoming README

### PR DESCRIPTION
**Pull request by bedroesb**
This will close #12. @martin-nc or @LianeHughes as natives have a look too ? ;)

### CODE REVIEW

**Comment by LianeHughes** [comment]
Happy to take a look, will try to do so today

File: README.md [hunk 1]
```diff
@@ -1,50 +1,46 @@
...
-1. deploy website
 
-    ```
-    bundle exec jekyll serve
-    ```
+<div style="padding:30px;"><p><i>Infectious Diseases Toolkit. A deliverable from the EU-funded BY-COVID project (grant agreement number 101046203). URL: <a href="https://www.infectious-diseases-toolkit.org/">https://www.infectious-diseases-toolkit.org/</a>.</i></p>
+</div>
 
-Additional information can be found at the following link: [https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll)
+The Infectious Diseases Toolkit makes all of their materials publicly available under Open Source Initiative licenses. The process documents and data are made available under a CC-BY license. Software are made available under an MIT license. For full details on licensing, please visit our License document.
```
Comment (LianeHughes) [inline README.md:42]:
```suggestion
All of the material from the IDTk is made publicly available under Open Source Initiative licences. The process documents and data are made available under a CC-BY license. Software is made available under an MIT licence. For full details on licensing, please see our [licence document](link to licence).
```

File: README.md [hunk 2]
```diff
@@ -1,50 +1,46 @@
...
+
+## Acknowledgements
+
+The Infectious Diseases Toolkit is developed in BY-COVID that received funding from the European Union’s Horizon Europe Research and Innovation Programme under grant agreement number 101046203.It is built based on the experience and ways of working of the [RDMkit](https://rdmkit.elixir-europe.org/) and also makes use of the [ELIXIR Toolkit Theme](https://elixir-belgium.github.io/elixir-toolkit-theme/) for deployment.
```
Comment (LianeHughes) [inline README.md:46]:
```suggestion
The IDTk is developed as part of the project BY-COVID, which received funding from the European Union’s Horizon Europe Research and Innovation Programme (grant number: 101046203). It is built based on experience and ways of working from the [RDMkit](https://rdmkit.elixir-europe.org/), and makes use of the [ELIXIR Toolkit Theme](https://elixir-belgium.github.io/elixir-toolkit-theme/) for deployment.
```

**Review by LianeHughes — COMMENTED** [review]
Here are my comments/thougts/suggestions

File: README.md [hunk 3]
```diff
@@ -1,50 +1,46 @@
...
 
-### Run using Docker
+## How to reach us
 
-If not already installed on your machine, install Docker. From the root of the ``infectious-diseases-toolkit`` directory, run:
-```
-docker run -it --rm -p 4000:4000 -v $PWD:/srv/jekyll jekyll/jekyll:latest /bin/bash -c "chmod a+w /srv/jekyll/Gemfile.lock && chmod 777 /srv/jekyll && jekyll serve -w"
-```
-This will start the docker container and serve the website locally.
+Do you have any question, a content proposal or see a mistake?
```
[the remark this answers is not in the document]
Comment (bedroesb) [reply README.md:19]: hmmm I wonder if it also covers 'I just want to file a problem', like: This link is not working.

**Comment by bedroesb** [comment]
@martin-nc and @LianeHughes I think I implemented the feedback, do let me know If something is off! If not, please prove and merge!

**Review by martin-nc — APPROVED** [review]
Looks okay. Hope @LianeHughes is okay with it, too.

**Pull request merged**